plugins/channel_header_plug/channel_header_plug.jsx

```diff
@@ -169,27 +186,26 @@ export default class ChannelHeaderPlug extends React.PureComponent {
                             overlay={this.state.dropdownOpen ? <></> : (
                                 <Tooltip id='removeIcon'>
                                     <div aria-hidden={true}>
-                                        <FormattedMessage
-                                            id='generic_icons.plugins'
-                                            defaultMessage='Plugins'
-                                        />
+                                        {this.props.tooltipMessage}
                                     </div>
                                 </Tooltip>
                             )}
                         >
-                            <React.Fragment>
-                                <PluginChannelHeaderIcon
-                                    id='pluginChannelHeaderIcon'
-                                    className='icon icon--standard icon__pluginChannelHeader'
-                                    aria-hidden='true'
-                                />
-                                <span
-                                    id='pluginCount'
-                                    className='icon__text'
-                                >
-                                    {plugs.length}
-                                </span>
-                            </React.Fragment>
+                            <div>
+                                <React.Fragment>
```

Comment:
You don't need the `React.Fragment` that is used to represent nodes that have no specific parent, in this case, you have a `<div>` that acts like the parent of all the children, so, you don't need it.